sql/core/src/main/scala/org/apache/spark/sql/execution/WholeStageCodegenExec.scala

```diff
@@ -679,6 +700,8 @@ case class WholeStageCodegenExec(child: SparkPlan)(val codegenStageId: Int)
       new CodeAndComment(CodeFormatter.stripExtraNewLines(source), ctx.getPlaceHolderToComments()))
 
     logDebug(s"\n${CodeFormatter.format(cleanedSource)}")
+    val duration = System.nanoTime() - startTime
```

Comment:
shall we do it before the debug? Otherwise we may include `CodeFormatter.format`

Reply:
OK.